bin/predict.py

```diff
@@ -75,37 +76,40 @@
     if args.model not in list(string.ascii_lowercase)[:11]:
         raise ValueError(f"Model only range from a -- j.")
 
-    cands_to_eval = glob.glob(f"{args.data_dir}/*h5")
-
-    if len(cands_to_eval) == 0:
-        raise FileNotFoundError(f"No candidates to evaluate.")
-
-    logging.debug(f"Read {len(cands_to_eval)} candidates")
-
-    # Get the data generator, make sure noise and shuffle are off.
-    cand_datagen = DataGenerator(
-        list_IDs=cands_to_eval,
-        labels=[0] * len(cands_to_eval),
-        shuffle=False,
-        noise=False,
-        batch_size=args.batch_size,
-    )
-
     model = get_model(args.model)
+    
+    for data_dir in args.data_dir:
+
+        cands_to_eval = glob.glob(f"{data_dir}/*h5")
+
+        if len(cands_to_eval) == 0:
+            raise FileNotFoundError(f"No candidates to evaluate.")
```

Comment:
If one of the directories is empty this would raise an error and exit the loop. Should we add a `logging.warning` and skip to the next directory instead?

Reply:
Yeah, that would probably be a better option for that case. The important thing is to print some separation indicators so that the message can be distinguished from the many that are printed in the process.

Reply:
I think `logging.warning` would take care of that.